iz_importer.py

```diff
@@ -179,9 +176,12 @@ def extract_copyright_from_string(self, copyright_string):
             copyright = os.path.splitext(copyright)[0]
         return copyright
 
+
```

Comment:
I wasn't sure how you intended to use metadatatools in this line, so I used the read exif metadata method as a placeholder, read over the methods in the metadata class and what kind of data structure they return to properly modify the code.

Reply:
copy, yuo can jkust comment this out entirely, I have a large delta incoming.